lldb/source/Plugins/Language/Swift/SwiftHighlighter/include/SwiftHighlighterSupport.h

```diff
@@ -0,0 +1,182 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors.
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+// FIXME: this is copied from include/lldb/Utility/AnsiTerminal.h, with
+// some modifications to remove references to llvm types (for example,
+// replacing llvm::StringRef with std::string), because at the moment
+// Swift/C++ interop is not able to import the LLDB (or LLVM) umbrella. This
+// should be removed when that's possible.
+
+#ifndef SWIFTHIGHLIGHTERSUPPORT_H
+#define SWIFTHIGHLIGHTERSUPPORT_H
+
+#define ANSI_FG_COLOR_BLACK 30
```

Comment:
These probably shouldn't be #defines anyway. Could you factor these enums out into a new header upstream that you _can_ import into Swift?